data/elan-42ab.tablet

```diff
@@ -8,13 +7,12 @@
 
 [Device]
 Name=Dell Inc. Inspiron 14 7445 2-in-1
-ModelName=
-DeviceMatch=i2c|04f3|42ab
+ModelName=ELAN3233:00 04F3:42AB
+DeviceMatch=i2c|04f3|42ab;i2c|04f3|42ab Stylus
```

Comment:
this one looks wrong, the third entry is the PID but you're using a string here. And for devices that have a unique PID we don't need a name entry anyway, so you can skip this change.